core/server/worker/src/main/java/alluxio/worker/block/meta/StorageTier.java

```diff
@@ -227,6 +231,13 @@ public StorageDir getDir(int dirIndex) {
     return Collections.unmodifiableList(mDirs);
   }
 
+  /**
+   * @return a list of lost storage paths
+   */
+  public List<String> getLostStorage() {
+    return Collections.unmodifiableList(mLostStorage);
```

Comment:
seems susceptible to concurrent modification exceptions. Could we return a copy instead?

Reply:
Changed to return a copy, thanks